### DISCUSSION TITLE
Is there a way to view /cancel running backend tasks?

### DISCUSSION THREAD

**Discussion by yellowbean** [reflex-dev/reflex#3097] [Q&A]
Hi, all, 

thanks for all the great work

I'm wondering , if in a web tab, I started a long-run backend task, is there a way to view all the long running tasks and then preferably, an ID associated with each of the long-run tasks, user then has the option to kill these long running tasks ?

**Comment by picklelo** [comment]
@yellowbean Check out the [background task docs](https://reflex.dev/docs/events/background-events) it shows how you can pause and check the status of tasks. You would have to add state vars that you update to set the task progress, which you can then check on the frontend and in other event handlers.

**Reply by yellowbean** [reply]
looks like the link is 404

**Reply by picklelo** [reply]
Sorry, updated the link to the correct URL
